### PR TITLE
Feature/snip24 example

### PR DESCRIPTION
**Pull request by levackt**
- Updated the package to latest secretjs
- Added a SNIP24 token example using query permits 
- Updated explorer URLs to ping pub
- Pulsar-2 to pulsar-3

### CODE REVIEW

File: 9_snip24_token/main.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+  // Create a connection to Secret Network node
+  // Pass in a wallet that can sign transactions
+  // Docs: https://github.com/scrtlabs/secret.js#secretnetworkclient
+  const txEncryptionSeed = EncryptionUtilsImpl.GenerateNewSeed();
+  const secretjs = new SecretNetworkClient({
+    url: process.env.SECRET_LCD_URL,
+    wallet: wallet,
+    walletAddress: wallet.address,
+    chainId: process.env.SECRET_CHAIN_ID,
+    txEncryptionSeed: txEncryptionSeed
```
Comment (assafmo) [inline 9_snip24_token/main.js:20]: This is unnecessary if not reused in a later session

**Review by assafmo — APPROVED** [review]
LGTM! Is there a reason for using SNIP-24 rather than the latest one? I think that HEAD is SNIP-25, we just haven't tagged it.

- https://github.com/SecretFoundation/SNIPs/blob/master/SNIP-25.md
- https://github.com/scrtlabs/snip20-reference-impl

**Comment by levackt** [comment]
No reason, will update to SNIP-25 thanks

**Comment by levackt** [comment]
Since it's non-blocking I'll update to 25 in a new PR, thanks @assafmo

File: 9_snip24_token/main.js (hunk 1 above)
Comment (levackt) [reply 9_snip24_token/main.js:20]: Got it

**Pull request merged**